Sources/CoffeeBot/CoffeeManager/CoffeeManager.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  CoffeeManager.swift
+//  SwiftBot
+//
+
+import Foundation
+import BotsKit
+import LoggerAPI
+
+internal protocol CoffeeManagerProtocol {
+    func process(activity:Activity)
+}
+
+class CoffeeManager {
```

Comment:
Ok, I think you can merge this class to CoffeeBot. And then extend CoffeeBot with Bot protocol.
So you have entry point to your Bot and you can test it as you want and in same time extending from Bot protocol give you ability to integrate it inside BotsKit